### ISSUE TITLE
Sails-CLI should look for an app.js in the local directory first.

### ISSUE THREAD

**Issue by dcbartlett** [balderdashy/sails#2926]
What are you guy's thoughts on having ```sails lift``` look for a local app.js file in the project and running that instead of the default.  This could be useful.  I know that Bootstrap allows you to add code before sails is lifted, but there may be other things that you need to override or do during the boot process that bootstrap doesn't allow access to.

**Comment by tjwebb** [comment]
I like this idea. We've discussed this internally quite a bit, actually. Running `node app.js` and `sails lift` shouldn't run two different programs, IMO.

**Comment by dcbartlett** [comment]
I can create a pull request on this if we get a few up-votes.  I'm actually working to implement this as i need the functionality.

**Comment by brandtheisey** [comment]
:+1:  I like this idea

**Comment by jeffjewiss** [comment]
Has there been any progress on this? I'd like to help out since it seems like a fairly small thing to implement.

**Comment by sailsbot** [comment]
Thanks for posting, @dcbartlett.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
